### PR TITLE
[NetAppFiles] Anf 6288 net app files 2020 06 01

### PR DESCRIPTION
**Pull request by audunn**
Bumps .net SDK to 2020-06-01 API version

### CODE REVIEW

File: sdk/netapp/Microsoft.Azure.Management.NetApp/src/Microsoft.Azure.Management.NetApp.csproj [hunk 1]
```diff
@@ -9,15 +9,20 @@
...
-
+        Version 1.7.0 relates to NetApp Files (ANF) 2020-06-01.
+          - Added support for Azure NetAppFiles data protection features snapshotpolicy, backuppolicy and backups.  
+          - Added BreakReplicationRequest body property to BreakReplication operation to force break a replication
+          - Added reinitializeReplication. Re-Initializes the replication connection on the destination volume
+					- Added PoolChange operation to volumes. Moves volume to another pool
+					- Added status enum to ActiveDirectory
+					- Added adName, kdcIP, serverRootCACertificate optional parameters to Activedirectory.
+					- Added totalThroughputMibps, utilizedThroughputMibps, qosType to Pool properties
+					-
```
Comment (AlexGhiondea) [inline sdk/netapp/Microsoft.Azure.Management.NetApp/src/Microsoft.Azure.Management.NetApp.csproj:24]: Extra `-` ?

**Review by AlexGhiondea — APPROVED** [review]

File: sdk/netapp/Microsoft.Azure.Management.NetApp/src/Microsoft.Azure.Management.NetApp.csproj (hunk 1 above)
Comment (audunn) [reply sdk/netapp/Microsoft.Azure.Management.NetApp/src/Microsoft.Azure.Management.NetApp.csproj:24]: will remove

**Comment by audunn** [comment]
@nisha-bhatia  can we move this along?

**Review by nisha-bhatia — APPROVED** [review]

**Comment by audunn** [comment]
This has two approvals, thank you btw,  can we get this merged?

**Pull request merged**